DeepSpeech.ipynb

```diff
@@ -106,7 +107,14 @@
     "do_fulltrace = bool(len(os.environ.get('ds_do_fulltrace', '')))\n",
     "\n",
     "if do_fulltrace:\n",
-    "    check_cupti()"
+    "    check_cupti()\n",
+    "\n",
+    "track_gpu_usage = bool(len(os.environ.get('ds_track_gpu_usage', '')))\n",
+    "gpu_usage_charts = os.environ.get('ds_gpu_usage_charts', 'gpu_usage_%%s_%d.png' % int(time.time()))\n",
+    "\n",
+    "if track_gpu_usage:\n",
+    "    gu = GPUUsage()\n",
+    "    gu.start()"
```

Comment:
Sorry, I had a look at the link and all I could read was related to producer/consumer files related to the computation itself. Since in our case, we just instantiate a new thread that calls subprocess.Popen() to gather output of nvidia-smi and put that into a CSV file, I don't see how that holds.